openmls/src/tree/secret_tree.rs

```diff
@@ -51,23 +51,18 @@ impl From<&MlsPlaintext> for SecretType {
 pub(crate) fn derive_tree_secret(
```

Comment:
Looks like this is just an `kdf_expand_label` now and we can remove this.

Reply:
Given that `DeriveTreeSecret` is explicitly defined in the spec before and after the spec change PR, I don't think it's a bad strategy to keep this.
Obviously it doesn't do anything functionally and should probably be `#[inline]` or so.

Reply:
I tend to agree with @raphaelrobert. I added an `#[inline]`, although I'm not sure it's going to do much, as well as a comment indicating that this function corresponds to the `DeriveTreeSecret` function. The latter might help later when correlating spec and implementation. @franziskuskiefer do you have a strong opinion on this?

Reply:
Not really. I'd rather remove it in the spec than keeping it here. But I wouldn't block on this.

Reply:
Then I'd say keep it in for now and file an issue with the spec. The secret tree needs a rewrite (or at least a major refactoring) anyway at some point and we can reconsider then if we feel like it's unnecessary.